apps/codeforafrica/src/components/NavBarNavList/index.js

```diff
@@ -0,0 +1,72 @@
+import { Link } from "@commons-ui/next";
+import SvgIcon from "@mui/material/SvgIcon";
+import PropTypes from "prop-types";
+import React from "react";
+
+import TwitterIcon from "@/codeforafrica/assets/twitterDesktop.svg";
```

Comment:
```suggestion
import TwitterIcon from "@/codeforafrica/assets/icons/Type=twitter, Size=24, Color=CurrentColor.svg";
```

Is already modified to use `currentColor` for stroke and will be white in mobile and black in desktop.